app/models/version.rb

```diff
@@ -0,0 +1,2 @@
+class Version < ActiveRecord::Base
```

Comment:
If I remove it I get a LoadError: "Unable to autoload constant Version". Thoughts?